packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3922,6 +3947,9 @@ export type OnRenderAvatarCallback = (
 userId?: string, options?: CustomAvatarOptions,
 defaultOnRender?: (props: CustomAvatarOptions) => JSX.Element) => JSX.Element | undefined;
 
+// @beta
+export const onResolveDNSDependencyLazy: () => Promise<DeepNoiseSuppressionEffectDependency>;
```

Comment:
Don't shorten here to DNS as DNS isn't used anywhere else, the name will be very long but `onResolveDeepNoiseSuppressionDependencyLazy`

Reply:
done